granian/log.py

```diff
@@ -23,6 +24,7 @@ class LogLevels(str, Enum):
     LogLevels.warn: logging.WARN,
     LogLevels.info: logging.INFO,
     LogLevels.debug: logging.DEBUG,
+    LogLevels.notset: logging.NOTSET,
 }
```

Comment:
We can remove `log_levels_map` at all and use the LogLevels enum above as a mapping. Let me know if you are interested.